app/views/foods/index.html.erb

```diff
@@ -3,7 +3,7 @@
   <h1 style='color: rgb(48, 221, 48);'>Welcome back!</h1>
   <h3>This is what you currently have in your pantry</h3>
   <div class='add-ingredient-button-div'>
-    <%= button_to 'Add ingredient', new_food_path, class: 'btn', method: :get %>
+    <%= button_to 'Add Food', new_food_path, class: 'btn', method: :get %>
```

Comment:
- [optional] Great job renaming the button. Please kindly add the size a little and the button text bolder so it's more visible and follows the design.